clang-tools-extra/clangd/ConfigFragment.h

```diff
@@ -341,6 +341,14 @@ struct Fragment {
     ///   Delimiters: empty pair of delimiters "()" or "<>"
     ///   FullPlaceholders: full name of both type and parameter
     std::optional<Located<std::string>> ArgumentLists;
+    /// Add #include directives when accepting code completions. Config
+    /// equivalent of the CLI option '--header-insertion'
+    /// Valid values are enum Config::HeaderInsertionPolicy values:
+    ///   "IWYU": Include what you use. Insert the owning header for top-level
+    ///     symbols, unless the header is already directly included or the
+    ///     symbol is forward-declared
+    ///   "NeverInsert": Never insert headers
```

Comment:
looks like there's a slight discrepancy here; value is documented as `NeverInsert` here, but config-compiler expects `Never`. Can you amend one of those?

Reply:
Whoops, my mistake! Do I make a new PR or push a new amend-ed commit to the existing branch?

Reply:
I think a new PR/branch is preferred (I am not an expert on how github PRs work though, hence I'd prefer the most straight-forward approach :D)

Reply:
Me neither :D

Will create a new PR with the proposed fix

Reply:
Created a PR at #135921